config/crd/kustomization.yaml

```diff
@@ -8,12 +8,12 @@ resources:
 patchesStrategicMerge:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
```

Comment:
Should these comment lines be removed once the PR is ready to be merged? (i.e. cleanup)